finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -21,10 +21,14 @@ import scala.io.Source
  */
 private object ClassPath {
 
-  private val ignoredPackages = Seq(
+  private val defaultIgnoredPackages = Seq(
     "apple/", "ch/epfl/", "com/apple/", "com/oracle/",
     "com/sun/", "java/", "javax/", "scala/", "sun/", "sunw/")
 
+  private[util] def ignoredPackages =
+    if (ignoredPaths.isDefined) defaultIgnoredPackages ++ ignoredPaths()
```

Comment:
the if statement isn't necessary.  we can just add ignoredPaths without checking.